http/src/http.ml

```diff
@@ -720,6 +720,14 @@ let is_keep_alive version headers =
 let pp_field field_name pp_v fmt v =
   Format.fprintf fmt "@[<1>%s:@ %a@]" field_name pp_v v
 
+let content_length (requires_content_length : 'a -> bool) a headers =
```

Comment:
```suggestion
let content_length (requires_content_length : 'a -> bool) a headers =
```
Why not simply bool?

Reply:
Indeed. Good catch indeed! I seem to have got the tunnel vision when doing these changes. It is `bool` now.